aboutpage.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apex Legends: About page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="background-cont-about">
+        <header>
+            <nav class="nav-desk">
+              <img class="menu-icon none-icon" src="./images/apex-menu.png" alt="nav-menu">
+              <img class="none-icon" src="./images/apex-menu.png" alt="nav-menu">
```

Comment:
- [ ] Please 🙏🏽 add the `a` tag to your logo, for a better UX when the user clicks on the logo it should take the user back to the home page. 

> The logo in the header links to the home page.